### PR TITLE
Bump ob dep

### PR DESCRIPTION
**Pull request by Siddharth2207**
## Motivation
Bump orderbook dep, and update the composition to work with the new cli tooling. Get orderbook address and generate addOrder calldata from the cli itself.

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [x] unit-tested any new functionality
- [x] linked any relevant issues or PRs

### CODE REVIEW

File: src/lib/LibComposeOrder.sol [hunk 1]
```diff
@@ -11,11 +11,12 @@ library LibComposeOrders {
     function getComposedOrder(
```
Comment (hardyjosh) [inline src/lib/LibComposeOrder.sol:11]: are you actually using this?

File: src/lib/LibComposeOrder.sol [hunk 2]
```diff
@@ -30,20 +31,23 @@ library LibComposeOrders {
...
-        ffi[15] = scenario;
+        ffi[14] = "-c";
+        ffi[15] = settingsPath;
+        ffi[16] = "-s";
+        ffi[17] = scenario;
 
         composedOrder = vm.ffi(ffi);
     }
 
     function getComposedPostAddOrder(
```
Comment (hardyjosh) [inline src/lib/LibComposeOrder.sol:42]: are you using this?

File: src/lib/LibComposeOrder.sol (hunk 1 above)
Comment (Siddharth2207) [reply src/lib/LibComposeOrder.sol:11]: yes

File: src/lib/LibComposeOrder.sol (hunk 2 above)
Comment (Siddharth2207) [reply src/lib/LibComposeOrder.sol:42]: yes

**Review by hardyjosh — APPROVED** [review]

**Pull request merged**